readthedocs/rtd_tests/tests/test_doc_building.py

```diff
@@ -1153,7 +1153,10 @@ def assertArgsStartsWith(self, args, function_mock):
             if arg is not mock.ANY:
                 self.assertTrue(arg_mock.startswith(arg))
 
-    def test_install_core_requirements_sphinx(self):
+    @patch('readthedocs.projects.models.Project.checkout_path')
```

Comment:
This shouldn't be necessary if we remove

https://github.com/rtfd/readthedocs.org/blob/ebf0987fc8a739b47cd8c0ef9fb6c65be7af4ff4/readthedocs/doc_builder/python_environments.py#L37-L38

I didn't do it here because the changes would be bigger.